v1/src/main/java/com/google/cloud/teleport/spanner/TextImportPipeline.java

```diff
@@ -277,6 +291,7 @@ public interface Options extends PipelineOptions {
 
     @TemplateParameter.GcsWriteFolder(
         order = 16,
+        groupName = "Target",
```

Comment:
Same. This is not the target but an extra output. Leave ungrouped.

Reply:
Done